### PR TITLE
Update filename to match new 2f_85 xacro

### PR DESCRIPTION
**Pull request by eholum**
That macro was renamed in: https://github.com/PickNikRobotics/ros2_robotiq_gripper/commit/c5afab820c38f48584d2122d14cec628ee4cb910

I believe this reference also needed to be updated.

### CODE REVIEW

**Review by abake48 — APPROVED** [review]

File: robotiq_description/urdf/robotiq_gripper.urdf.xacro
[the remark this answers is not in the document]
Comment (eholum) [reply robotiq_description/urdf/robotiq_gripper.urdf.xacro:1]: Good call, updated. Does that work for you?

[the remark this answers is not in the document]
Comment (MarqRazz) [reply robotiq_description/urdf/robotiq_gripper.urdf.xacro:1]: I don't see the change. I expected this file name to be updated to `robotiq_description/urdf/robotiq_2f_85_gripper.urdf.xacro`

[the remark this answers is not in the document]
Comment (eholum) [reply robotiq_description/urdf/robotiq_gripper.urdf.xacro:1]: I think GitHub is borked... Not sure why it's not rendering in this PR but it's on the branch! https://github.com/PickNikRobotics/ros2_robotiq_gripper/compare/main...eholum:ros2_robotiq_gripper:pr-fix-gripper-reference

EDIT: Oh - degraded availability: https://www.githubstatus.com

**Comment by eholum** [comment]
Ok, @MarqRazz @abake48 did the new change show up? A little disconcerting that it didn't dismiss your review but I'll chalk that off to GitHub games.

**Comment by abake48** [comment]
> Ok, @MarqRazz @abake48 did the new change show up? A little disconcerting that it didn't dismiss your review but I'll chalk that off to GitHub games.

I'm seeing the change now

**Review by MarqRazz — APPROVED** [review]

**Pull request merged**